Client/src/components/layout/Links/Nav/Nav.modules.css

```diff
@@ -0,0 +1,19 @@
+.Nav {
+  list-style: none;
+}
+
+.Nav a {
+  text-decoration: none;
+  padding: 10px 8px;
+  color: inherit;
+  margin: 0 4px;
+  border-radius: 4px;
+  transition: 180ms;
+}
+
+.dark .Nav a:hover {
```

Comment:
Can you elaborate why some class selectors start with capital letter but some are not?